src/Api/SecretsManager/Models/Response/ProjectCountsResponseModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Bit.Core.Models.Api;
+
+namespace Bit.Api.SecretsManager.Models.Response;
+
+public class ProjectCountsResponseModel() : ResponseModel(_objectName)
+{
+    private const string _objectName = "project_counts";
+
+
+    public int Secrets { get; set; }
+
+    public int People { get; set; }
+
+    public int MachineAccounts { get; set; }
```

Comment:
```suggestion
    public int ServiceAccounts { get; set; }
```

Reply:
Fixed in https://github.com/bitwarden/server/commit/41b166c6059a33ccd52066d0b524f387254ffa0d